SECURITY.md

```diff
@@ -1,24 +1,29 @@
 <!--
   Licensed to the Apache Software Foundation (ASF) under one
-  or more contributor license agreements.  See the NOTICE file
+  or more contributor license agreements. See the NOTICE file
   distributed with this work for additional information
-  regarding copyright ownership.  The ASF licenses this file
+  regarding copyright ownership. The ASF licenses this file
   to you under the Apache License, Version 2.0 (the
   "License"); you may not use this file except in compliance
-  with the License.  You may obtain a copy of the License at
+  with the License. You may obtain a copy of the License at
 
    http://www.apache.org/licenses/LICENSE-2.0
 
   Unless required by applicable law or agreed to in writing,
   software distributed under the License is distributed on an
   "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
-  KIND, either express or implied.  See the License for the
+  KIND, either express or implied. See the License for the
   specific language governing permissions and limitations
   under the License.
 -->
 
-# Security Policy
+# Apache Gravitino Security Policy
 
-The Apache Gravitino project follows the standard processes as set out by the [ASF security team](https://www.apache.org/security/).
+The Apache Gravitino project adheres to the security protocols and standards established by the Apache Software Foundation (ASF) Security Team.
```

Comment:
You've changed the meaning to include security protocols, that's not the intent of this document. The link should also be left in.